src/test/java/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqFilterSparkIntegrationTest.java

```diff
@@ -1,94 +1,99 @@
 package org.broadinstitute.hellbender.tools.spark.pathseq;
 
 import htsjdk.samtools.ValidationStringency;
+import htsjdk.samtools.metrics.MetricsFile;
 import org.broadinstitute.hellbender.CommandLineProgramTest;
 import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
 import org.broadinstitute.hellbender.utils.test.SamAssertionUtils;
 import org.testng.Assert;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import java.io.File;
 
 public class PathSeqFilterSparkIntegrationTest extends CommandLineProgramTest {
 
+    static final String imagePath = "src/test/resources/" + PathSeqFilterSpark.class.getPackage().getName().replace(".", "/") + "/hg19mini.fasta.img";
```

Comment:
I believe there is a constant for "src/test/resouces" perhaps something like ```publicTestResourcesDir``` (yes in lowercase :( )

Reply:
Not sure of the name but I'm quite sure there is one.

Reply:
Done